internal/operators/ocs/validations.go

```diff
@@ -65,7 +66,7 @@ func setOperatorStatus(cluster *models.Cluster, status string) error {
 }
 
 // ValidateOCSRequirements is used to validate min requirements of OCS
-func (o *ocsValidator) ValidateOCSRequirements(cluster *models.Cluster) string {
+func (o *ocsValidator) ValidateOCSRequirements(cluster *models.Cluster) (api.ValidationStatus, string) {
```

Comment:
Should this method be renamed to just ValidateRequirements ?

Reply:
Good idea! All the methods are by definition for OCS, so having "OCS"  in their names is redundant; will rename.

Reply:
Done